src/i18n/strings/en_EN.json

```diff
@@ -927,5 +927,9 @@
     "%(serverName)s Matrix ID": "%(serverName)s Matrix ID",
     "Add rooms to this group": "Add rooms to this group",
     "Invites sent": "Invites sent",
-    "Your group invitations have been sent.": "Your group invitations have been sent."
+    "Your group invitations have been sent.": "Your group invitations have been sent.",
```

Comment:
Does it make sense to `s/group/community` as part of this PR? (while you're here)

Reply:
I'd rather keep the two things separate tbh.